<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>frmConfigEnable</class>
 <widget class="QWidget" name="frmConfigEnable">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="spacing">
    <number>6</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QListWidget" name="listWidget1">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Ignored">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <item>
      <property name="text">
       <string>x分屏</string>
      </property>
      <property name="checkState">
       <enum>Unchecked</enum>
      </property>
     </item>
     <item>
      <property name="text">
       <string>y分屏</string>
      </property>
      <property name="checkState">
       <enum>Unchecked</enum>
      </property>
     </item>
     <item>
      <property name="text">
       <string>1分屏</string>
      </property>
      <property name="checkState">
       <enum>Unchecked</enum>
      </property>
     </item>
     <item>
      <property name="text">
       <string>4分屏</string>
      </property>
      <property name="checkState">
       <enum>Unchecked</enum>
      </property>
     </item>
     <item>
      <property name="text">
       <string>6分屏</string>
      </property>
      <property name="checkState">
       <enum>Unchecked</enum>
      </property>
     </item>
     <item>
      <property name="text">
       <string>8分屏</string>
      </property>
      <property name="checkState">
       <enum>Unchecked</enum>
      </property>
     </item>
     <item>
      <property name="text">
       <string>9分屏</string>
      </property>
      <property name="checkState">
       <enum>Unchecked</enum>
      </property>
     </item>
     <item>
      <property name="text">
       <string>13分屏</string>
      </property>
      <property name="checkState">
       <enum>Unchecked</enum>
      </property>
     </item>
     <item>
      <property name="text">
       <string>16分屏</string>
      </property>
      <property name="checkState">
       <enum>Unchecked</enum>
      </property>
     </item>
     <item>
      <property name="text">
       <string>25分屏</string>
      </property>
      <property name="checkState">
       <enum>Unchecked</enum>
      </property>
     </item>
     <item>
      <property name="text">
       <string>36分屏</string>
      </property>
      <property name="checkState">
       <enum>Unchecked</enum>
      </property>
     </item>
     <item>
      <property name="text">
       <string>64分屏</string>
      </property>
      <property name="checkState">
       <enum>Unchecked</enum>
      </property>
     </item>
    </widget>
   </item>
   <item>
    <widget class="QListWidget" name="listWidget2">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Ignored">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <item>
      <property name="text">
       <string>视频回放</string>
      </property>
      <property name="checkState">
       <enum>Unchecked</enum>
      </property>
     </item>
     <item>
      <property name="text">
       <string>电子地图</string>
      </property>
      <property name="checkState">
       <enum>Unchecked</enum>
      </property>
     </item>
     <item>
      <property name="text">
       <string>日志查询</string>
      </property>
      <property name="checkState">
       <enum>Unchecked</enum>
      </property>
     </item>
     <item>
      <property name="text">
       <string>备用模块</string>
      </property>
      <property name="checkState">
       <enum>Unchecked</enum>
      </property>
     </item>
     <item>
      <property name="text">
       <string>图片地图</string>
      </property>
      <property name="checkState">
       <enum>Unchecked</enum>
      </property>
     </item>
     <item>
      <property name="text">
       <string>网络回放</string>
      </property>
      <property name="checkState">
       <enum>Unchecked</enum>
      </property>
     </item>
     <item>
      <property name="text">
       <string>远程回放</string>
      </property>
      <property name="checkState">
       <enum>Unchecked</enum>
      </property>
     </item>
     <item>
      <property name="text">
       <string>图片回放</string>
      </property>
      <property name="checkState">
       <enum>Unchecked</enum>
      </property>
     </item>
     <item>
      <property name="text">
       <string>视频上传</string>
      </property>
      <property name="checkState">
       <enum>Unchecked</enum>
      </property>
     </item>
     <item>
      <property name="text">
       <string>设备日志</string>
      </property>
      <property name="checkState">
       <enum>Unchecked</enum>
      </property>
     </item>
     <item>
      <property name="text">
       <string>轮询配置</string>
      </property>
      <property name="checkState">
       <enum>Unchecked</enum>
      </property>
     </item>
     <item>
      <property name="text">
       <string>用户管理</string>
      </property>
      <property name="checkState">
       <enum>Unchecked</enum>
      </property>
     </item>
     <item>
      <property name="text">
       <string>录像计划</string>
      </property>
      <property name="checkState">
       <enum>Unchecked</enum>
      </property>
     </item>
     <item>
      <property name="text">
       <string>其他配置</string>
      </property>
      <property name="checkState">
       <enum>Unchecked</enum>
      </property>
     </item>
     <item>
      <property name="text">
       <string>天气预报</string>
      </property>
      <property name="checkState">
       <enum>Unchecked</enum>
      </property>
     </item>
     <item>
      <property name="text">
       <string>视频工具栏</string>
      </property>
      <property name="checkState">
       <enum>Unchecked</enum>
      </property>
     </item>
     <item>
      <property name="text">
       <string>标题栏菜单</string>
      </property>
      <property name="checkState">
       <enum>Unchecked</enum>
      </property>
     </item>
     <item>
      <property name="text">
       <string>子模块菜单</string>
      </property>
      <property name="checkState">
       <enum>Unchecked</enum>
      </property>
     </item>
     <item>
      <property name="text">
       <string>子模块移动</string>
      </property>
      <property name="checkState">
       <enum>Unchecked</enum>
      </property>
     </item>
    </widget>
   </item>
   <item>
    <widget class="QListWidget" name="listWidget3">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Ignored">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <item>
      <property name="text">
       <string>分辨率</string>
      </property>
      <property name="checkState">
       <enum>Unchecked</enum>
      </property>
     </item>
     <item>
      <property name="text">
       <string>帧率</string>
      </property>
      <property name="checkState">
       <enum>Unchecked</enum>
      </property>
     </item>
     <item>
      <property name="text">
       <string>设备地址</string>
      </property>
      <property name="checkState">
       <enum>Unchecked</enum>
      </property>
     </item>
     <item>
      <property name="text">
       <string>封装格式</string>
      </property>
      <property name="checkState">
       <enum>Unchecked</enum>
      </property>
     </item>
     <item>
      <property name="text">
       <string>视频解码格式</string>
      </property>
      <property name="checkState">
       <enum>Unchecked</enum>
      </property>
     </item>
     <item>
      <property name="text">
       <string>音频解码格式</string>
      </property>
      <property name="checkState">
       <enum>Unchecked</enum>
      </property>
     </item>
     <item>
      <property name="text">
       <string>实时码率</string>
      </property>
      <property name="checkState">
       <enum>Unchecked</enum>
      </property>
     </item>
     <item>
      <property name="text">
       <string>硬解名称</string>
      </property>
      <property name="checkState">
       <enum>Unchecked</enum>
      </property>
     </item>
     <item>
      <property name="text">
       <string>视频模式</string>
      </property>
      <property name="checkState">
       <enum>Unchecked</enum>
      </property>
     </item>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
